assets/js/common/AdvisoryIcon/AdvisoryIcon.stories.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import AdvisoryIcon from '.';
+
+export default {
+  title: 'Components/AdvisoryIcon',
+  components: AdvisoryIcon,
+  argTypes: {
+    type: {
+      description: 'The type of advisory',
+      control: {
+        type: 'string',
```

Comment:
I think it's possible to indicate the enum values of `type` here, in Storybook, with a `radio` control.